Alignment/OfflineValidation/plugins/EopTreeWriter.cc

```diff
@@ -137,7 +137,7 @@ EopTreeWriter::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
    // get geometry
    edm::ESHandle<CaloGeometry> geometry;
    iSetup.get<CaloGeometryRecord>().get(geometry);
-   const CaloGeometry* geo = geometry.product();
+   CaloGeometry* geo = (CaloGeometry*)(geometry.product());
```

Comment:
Here and many other places: can we avoid C-style casts, please? (e.g. IIUC this sould become `const_cast<CaloGeomety*>(...)`.

Reply:
do we really have to move to non-const geometry?
This sounds dangerous and not thread safe if non-constness is real.